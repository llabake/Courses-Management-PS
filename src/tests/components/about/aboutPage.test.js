import React from 'react';
import { shallow } from 'enzyme';
import AboutPage from "../../../components/about/AboutPage";

describe('<AboutPage />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<AboutPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
