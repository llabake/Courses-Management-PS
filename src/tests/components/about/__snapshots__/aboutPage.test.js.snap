// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutPage /> renders correctly 1`] = `
<div>
  <h1>
    About
  </h1>
  <p>
    This application uses React, Redux, React Router and a variety of other helpful libraries
  </p>
</div>
`;
